backends/influx.py

```diff
@@ -135,21 +135,17 @@ def delete(self, namespace, start=None, end=None):
 
         if _validate_namespace(namespace):
             namespace, _ = _extract_field(namespace)
-        else:
-            log.error('Error in delete method due invalid namespace value.')
-            return
 
         if not self._namespace_exists(namespace):
-            log.error(f'Error to delete because namespace \'{namespace}\' does'
-                      'not exist.')
-            return
+            error = (f'Error to delete because namespace \'{namespace}\' does'
+                     'not exist.')
+            raise NamespaceNotExistsError
```

Comment:
Do we really need exceptions so specific? Maybe just raising `ValueError` would do it.